hazelcast/src/test/java/com/hazelcast/client/impl/ClientDelegatingFutureTest.java

```diff
@@ -117,7 +117,6 @@ public void joinInternal_whenCompletedExceptionally() {
         invocationFuture.completeExceptionally(new IllegalArgumentException());
 
         expected.expect(IllegalArgumentException.class);
-        expected.expectCause(new RootCauseMatcher(IllegalArgumentException.class));
```

Comment:
It seems we only test client futures behaviour with `RuntimeException` in this test. For member-side invocations, `Invocation_ExceptionTest` tests those cases. Can we add a similar test for client invocation future?